evoting/service/open_test.go

```diff
@@ -77,6 +78,22 @@ func TestOpen_Full(t *testing.T) {
 	master.GenChain(nil)
 
 	election := &lib.Election{}
+	_, err := s.Open(&evoting.Open{Token: token, ID: master.ID, Election: election})
+	assert.NotNil(t, err)
```

Comment:
Since `Open` might return other errors, could you check that `err.Error()` contains the "election cannot end before current time" string please?

Reply:
👍 added assertion in new commit